lib/exposes.js

```diff
@@ -647,5 +649,9 @@ module.exports = {
             .withFeature(new Enum('state', access.SET, ['system_is_armed', 'system_is_disarmed']).withDescription('Set Squawk state'))
             .withFeature(new Enum('level', access.SET, ['low', 'medium', 'high', 'very_high']).withDescription('Sound level'))
             .withFeature(new Binary('strobe', access.SET, true, false).withDescription('Turn on/off the strobe (light) for Squawk')),
+        schedule: () => new List('schedule', access.STATE_SET, new Composite('period', access.SET)
+            .withFeature(new Numeric('hours', access.SET))
+            .withFeature(new Numeric('minutes', access.SET))
+            .withFeature(new Numeric('temperature', access.SET))),
```

Comment:
@Koenkk I didn't clearly understand what you meant, but after inspecting the provided example, will this format be ok?

Reply:
This looks better. I missed to mention this earlier but could the schedule part from TV02-Zigbee be re-used?

https://github.com/Koenkk/zigbee-herdsman-converters/blob/a41e7cf0d78bea89997103a23a613c6c79607810/devices/tuya.js#L2144

https://github.com/Koenkk/zigbee-herdsman-converters/blob/a41e7cf0d78bea89997103a23a613c6c79607810/devices/tuya.js#L2186

Reply:
> I missed to mention this earlier but could the schedule part from TV02-Zigbee be re-used?

Partially, yes. I'll show.